demo/src/main/java/com/example/DemoApplication.java

```diff
@@ -33,6 +33,10 @@ public void run(String... arg0) throws Exception {
 				userRepository.findAll().stream().forEach(user -> System.out.println(user));
 
 				System.out.println(userRepository.findAllByName("Raj"));
+
+				// Intentional bug
+				if(true) {
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Change this condition so that it does not always evaluate to "true" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://192.168.99.100:9000/coding_rules#rule_key=squid%3AS2583)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either remove or fill this block of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://192.168.99.100:9000/coding_rules#rule_key=squid%3AS00108)